Project1/application.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+from flask import Flask, session,url_for,redirect
+from flask import Flask,render_template,request,flash
+from flask_session import Session
+from  datetime import datetime
+# from Database import *
+
+app = Flask(__name__)
+app.secret_key = "secret"
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+
+@app.route('/')
+def indexed():
+    if 'username' in session:
+        username = session['username']
+        return 'Logged in as ' + username + '<br>' + \
+         "<b><a href = '/logout'>click here to log out</a></b>"
+    return "You are not logged in <br><a href = '/login'></b>" + \
+      "click here to log in</b></a>"
+  
+
+@app.route("/register")
+def index():    
+    return render_template("Registration.html")
+
+@app.route("/User",methods = ["GET","POST"])
```

Comment:
route should be /register

Reply:
On POST request if user is not existant, user should be created else you should display user already exists. If it's a GET request the registration page must be displayed.